finagle-netty4/src/test/scala/com/twitter/finagle/netty4/Netty4SslTest.scala

```diff
@@ -0,0 +1,111 @@
+package com.twitter.finagle.netty4
+
+import java.net.{InetAddress, InetSocketAddress, SocketAddress}
```

Comment:
please sort the java.* with the rest of the imports.  we don't treat any imports specially.